api/app/signals/apps/api/v1/views/signal.py

```diff
@@ -49,6 +55,65 @@ def retrieve(self, request, signal_id):
         return Response(data)
 
 
+class PublicSignalListViewSet(PublicSignalGenericViewSet):
+    # django-drf has too much overhead with these kinds of 'fast' request.
+    # When implemented using django-drf, retrieving a large number of elements cost around 4s (profiled)
+    # Using pgsql ability to generate geojson, the request time reduces to 30ms (> 130x speedup!)
+    # The downside is that this query has to be (potentially) maintained when changing one of the
+    # following models: sginal, categoryassignment, category, location, status
+    def list(self, *args, **kwargs):
+        fast_query = f"""
+        select jsonb_build_object(
+            'type', 'FeatureCollection',
+            'features', json_agg(features.feature)
+        ) as result from (
+            select json_build_object(
+                'type', 'feature',
```

Comment:
According to spec, and various linters I tried, `'feature'` should be `'Feature'` here.

Reply:
fixed it in the interest of time